<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1442</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timecard</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icons/logo.svg</normaloff>../Icons/logo.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog{
	background-color: white;
}
</string>
  </property>
  <widget class="QTableWidget" name="TimeListTable">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>55</y>
     <width>1191</width>
     <height>610</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Poppins SemiBold</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Header Design */

QHeaderView::section {
	font-family: Poppins;
	font-weight: bold;
	font-size: 12px;
	color: white;
	background-color: &quot;#344273&quot;;
	border: none;
}

/* Style for the selected row */
QTableView::item:selected:active {
	 selection-background-color: &quot;#9DADDF&quot;;
}

/* Style for the alternate row */
QTableWidget {
	alternate-background-color: &quot;#DDDDDD&quot;;
}</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideRight</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>17</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>166</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>31</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>BioNum</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>EmpName</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Trans_Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>MachCode</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Check_In</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Check_Out</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sched_In</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sched_Out</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>REMARKS</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Late/UT</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hours Work</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hours Work 2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Allowed OT</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>OTNote</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Leave</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>LaeveNote</string>
    </property>
   </column>
  </widget>
  <widget class="QFrame" name="sideBar">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>0</y>
     <width>261</width>
     <height>670</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#sideBar{
	background-color: qlineargradient(spread:pad, x1:0.508, y1:0.0340909, x2:0.503051, y2:1, stop:0 rgba(33, 44, 89, 255), stop:0.497175 rgba(52, 66, 115, 255), stop:0.847458 rgba(72, 87, 142, 255))
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="searchBar">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#searchBar{
	border: 1px solid &quot;#CBD5E0&quot;;
	border-radius: 3px; 
 	background-color: white;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="searchBioNum">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>10</y>
       <width>180</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#txtSearch{

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Type to search...</string>
     </property>
    </widget>
    <widget class="QLabel" name="searchIcon">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#searchIcon{
	background-color: white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Icons/search.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>231</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnCheckSched, #btnFilter {
	background-color: &quot;#546aba&quot;;
	color: white;
	border-radius: 5px;
}

#btnTimeSheet, #btnImport{
border-radius: 5px;
	color: white;
background-color: &quot;#30d977&quot;;
}

#btnExport{
background-color: rgb(255, 110, 112);
	color: white;
	border-radius: 5px;
}

#btnCheckSched:hover, #btnFilter:hover{
background-color: #485ba1;
}

#btnTimeSheet:hover, #btnImport:hover{
	background-color: &quot;#26ad5f&quot;;
}

#btnExport:hover{
background-color: #c75657;
}

</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="btnFilter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>FILTER</string>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCheckSched">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>CHECK SCHEDULE</string>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTimeSheet">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>PROCESS TIMESHEET</string>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>IMPORT</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Icons/import.svg</normaloff>../Icons/import.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>EXPORT</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../Icons/export.svg</normaloff>../Icons/export.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="header">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#header{
background-color: &quot;#DCE5FE&quot;;
}

#yearCC, #dateToCC, #dateFromCC{
border: 1px solid black;
border-radius: 5px;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lblYear">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Year:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dateFromCC">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>11</y>
      <width>170</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="yearCC">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>11</y>
      <width>170</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
     </font>
    </property>
    <property name="iconSize">
     <size>
      <width>5</width>
      <height>5</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="dateToCC">
    <property name="geometry">
     <rect>
      <x>715</x>
      <y>11</y>
      <width>170</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblDateFrom">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Day From:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDateTo">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>11</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Day To:</string>
    </property>
   </widget>
  </widget>
  <zorder>header</zorder>
  <zorder>TimeListTable</zorder>
  <zorder>sideBar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
